VultisigApp/VultisigApp/View Models/SendTransaction.swift

```diff
@@ -117,20 +105,20 @@ class SendTransaction: ObservableObject, Hashable {
     }
     
     var amountInRaw: BigInt {
-        let decimals = Double(coin.decimals)
-        return BigInt(amountDecimal * pow(10, decimals))
+        let decimals = coin.decimals
+        let amountInDecimals = amountDecimal * pow(10, decimals)
```

Comment:
coin has `func raw(for value: Decimal) -> BigInt`

Reply:
This one does not apply bc of the precision.